index.html

```diff
@@ -29,25 +29,112 @@
             </div>
-            <div class="intro ">
-                <h1>Hey there. <br>I am Sediqullah Badakhsh</h1>
-                <h2>I am a Software Developer</h2>
-                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <br>
+            <div class="cards-section">
+                <div class="work-card">
+                    <div class="card-content">
+                        <h4>Professional Art Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="work-card">
+                    <div class="card-content">
+                        <h4>Professional Art Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="work-card">
+                    <div class="card-content">
+                        <h4>Professional Art Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
             </div>
-        </div>
-        <div>
-            <img class="landing-image " src="image/landig-image.jpg" alt="landing-image "> </div>
+        </section>
+        <section class="about-me">
+            <div class="about-text">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button">Get My Resume</button>
+                <hr class="hr">
+            </div>
```

Comment:
- [ ] Please consider making your design here to match that of your chosen design by using the correct margins, font-weight, and proper spacing and also aligning the about me section properly. 😄

Your design:
<img width="218" alt="Loom_eKHNlUvzQT" src="https://user-images.githubusercontent.com/81468675/148625452-8ffb3f31-ec68-4657-a513-7318cc5400cf.png">

Figma design:
<img width="210" alt="Loom_M0rC2nFCI9" src="https://user-images.githubusercontent.com/81468675/148625491-e563b92f-0c1f-4b69-a977-68ed8b27ec46.png">